promptulate/agents/tool_agent/agent.py

```diff
@@ -127,11 +127,12 @@ def _run(self, prompt: str, *args, **kwargs) -> str:
             if "Final Answer" in answer:
                 return answer.split("Final Answer:")[-1]
 
-            action, action_input = self._find_action(answer)
+            action, thought, action_input = self._find_action(answer)
```

Comment:
May be is function should be change to `_parse_llm_response(answer)`.

Reply:
I think so.